### ISSUE TITLE
phpdbg_*() functions are scoped to PHPUnit\phpdbg_*()

### ISSUE THREAD

**Issue by sebastianbergmann** [sebastianbergmann/phpunit#4211]
PHP-Scoper, for example, transforms this [line](https://github.com/sebastianbergmann/php-code-coverage/blob/8.0.1/src/Driver/PHPDBG.php#L32)
```php
if (!\function_exists('phpdbg_start_oplog')) {
```
to 
```php
if (!\function_exists('PHPUnit\\phpdbg_start_oplog')) {
```
which, of course, does not work properly anymore.

**Comment by sebastianbergmann** [comment]
@theofidry @sebastianfeldmann How do we fix this? Is this something missing on PHPUnit's side (as in how we use PHP-Scoper) or is this a problem in PHP-Scoper (as in it should know that `phpdbg_*()` functions are built into PHP)?

**Comment by theofidry** [comment]
There is two solutions for it:

- Adding those PHPDBG functions to PHP-Scoper's reflector (see https://github.com/humbug/php-scoper/blob/master/src/Reflector.php#L26)
- Adding a PHPDBG stub to [JetBrains' stubs](https://github.com/JetBrains/phpstorm-stubs)

Those two are not mutually exclusive, just potentially redundant.

Also note that there is a discussion to allow this part to be directly configurable from the config (see https://github.com/humbug/php-scoper/issues/303#issuecomment-591283618) which would make it even faster to patch (as a workaround – it doesn't prevent having to patch upstream for a "cleaner" solution)

**Comment by theofidry** [comment]
Ah right, meanwhile the 3rd quick & dirty alternative is to leverage the patchers to replace the incorrect replacements

**Comment by sebastianbergmann** [comment]
I have created pull requests for both [php-scoper](https://github.com/humbug/php-scoper/pull/391) and [phpstorm-stubs](https://github.com/JetBrains/phpstorm-stubs/pull/793) which (hopefully) teach these two about the `phpdbg_*()` functions. I would rather not use a patcher (that I have to remove later) to address this problem.

**Comment by theofidry** [comment]
👍 will try to go through my OSS notifs tomorrow

**Comment by sebastianbergmann** [comment]
https://github.com/JetBrains/phpstorm-stubs/pull/793 has been merged.

**Comment by sebastianbergmann** [comment]
@theofidry Any update on this? Thanks!

**Comment by theofidry** [comment]
Should be fixed in 0.13.2
